crates/cli/src/subcommands/call.rs

```diff
@@ -34,9 +36,35 @@ pub fn cli() -> clap::Command {
         .after_help("Run `spacetime help call` for more detailed information.\n")
 }
 
+enum CallResult<'a> {
```

Comment:
it looks like this is just a description of what the call result will be, not the result itself, is that right?